content/en/docs/setup/install/operator/snips.sh

```diff
@@ -112,80 +105,97 @@ spec:
 EOF
 }
 
-snip_update_3() {
+snip_operator_logs() {
 kubectl logs -f -n istio-operator "$(kubectl get pods -n istio-operator -lname=istio-operator -o jsonpath='{.items[0].metadata.name}')"
 }
 
-snip_inplace_upgrade_1() {
+snip_inplace_upgrade() {
 <extracted-dir>/bin/istioctl operator init
 }
 
-snip_inplace_upgrade_2() {
+snip_inplace_upgrade_get_pods_istio_operator() {
 kubectl get pods --namespace istio-operator \
   -o=jsonpath='{range .items[*]}{.metadata.name}{":\t"}{range .spec.containers[*]}{.image}{", "}{end}{"\n"}{end}'
 }
 
-snip_inplace_upgrade_3() {
+snip_inplace_upgrade_get_pods_istio_system() {
 kubectl get pods --namespace istio-system \
   -o=jsonpath='{range .items[*]}{"\n"}{.metadata.name}{":\t"}{range .spec.containers[*]}{.image}{", "}{end}{"\n"}{end}'
 }
 
-snip_canary_upgrade_1() {
-kubectl get iop --all-namespaces
+snip_download_istio_previous_version() {
+curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.17.0 sh -
 }
 
-! read -r -d '' snip_canary_upgrade_1_out <<\ENDSNIP
-NAMESPACE      NAME                        REVISION   STATUS    AGE
-istio-system   example-istiocontrolplane              HEALTHY   11m
-ENDSNIP
+snip_deploy_operator_previous_version() {
+istio-1.17.0/bin/istioctl operator init
+}
 
-snip_canary_upgrade_2() {
-istio-1.8.1/bin/istioctl operator init --revision 1-8-1
+snip_install_istio_previous_version() {
+kubectl apply -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+  name: example-istiocontrolplane-1-17-0
+spec:
+  profile: default
+EOF
 }
 
-snip_canary_upgrade_3() {
-helm install istio-operator manifests/charts/istio-operator \
-  --set watchedNamespaces=istio-system \
-  -n istio-operator \
-  --set revision=1-9-0
+snip_verify_operator_cr() {
+kubectl get iop --all-namespaces
 }
 
-snip_canary_upgrade_4() {
-cat example-istiocontrolplane-1-8-1.yaml
+! read -r -d '' snip_verify_operator_cr_out <<\ENDSNIP
+NAMESPACE      NAME                              REVISION   STATUS    AGE
+istio-system   example-istiocontrolplane1-17-0              HEALTHY   11m
+ENDSNIP
+
+snip_canary_upgrade_init() {
+istio-1.18.0/bin/istioctl operator init --revision 1-18-0
+}
+
+snip_cat_operator_yaml() {
+cat example-istiocontrolplane-1-18-0.yaml
 }
 
-! read -r -d '' snip_canary_upgrade_4_out <<\ENDSNIP
+! read -r -d '' snip_cat_operator_yaml_out <<\ENDSNIP
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 metadata:
   namespace: istio-system
-  name: example-istiocontrolplane-1-8-1
+  name: example-istiocontrolplane-1-18-0
 spec:
-  revision: 1-8-1
+  revision: 1-18-0
   profile: demo
```

Comment:
Don't change the profile during upgrade, both should be demo or both default.
```suggestion
  profile: default
```